lib/iex/lib/iex/helpers.ex

```diff
@@ -533,14 +533,37 @@ defmodule IEx.Helpers do
 
   defp print_runtime_info_topic(:memory) do
     print_pane("Memory")
-    print_memory("Total", :total, :MB)
+    print_memory("Total", :total)
     print_memory("Atoms", :atom)
     print_memory("Binaries", :binary)
     print_memory("Code", :code)
     print_memory("ETS", :ets)
     print_memory("Processes", :processes)
   end
 
+  defp print_runtime_info_topic(:memory_details) do
+    print_pane("Memory Details")
+    print_memory_detail("Atom Space", :atom_space)
+    print_memory_detail("Atom Table", :atom_table)
+    print_memory_detail("BIF Timer", :bif_timer)
+    print_memory_detail("Bits Bufs Size", :bits_bufs_size)
```

Comment:
Does `bits` here refers to bitstrings? :thinking:

Reply:
Ping @michalmuskala.

Reply:
Looking through the OTP code, it seems that it's something allocated when placing integers in binaries, when the binary is not byte-aligned.